.github/workflows/ruby_build.yml

```diff
@@ -0,0 +1,33 @@
+name: Ruby Build
+
+on:
+  release:
+    types: published
+
+env:
+  BUNDLE_WITHOUT: 'test lint'
```

Comment:
what about 'develop'?

Reply:
development includes the gems that are part of the asset pipeline, so are needed to build the javascript

Reply:
OK thanks. Why are they in development rather than default then?